internal/host/statemachine.go

```diff
@@ -233,6 +233,16 @@ func NewHostStateMachine(th *transitionHandler) stateswitch.StateMachine {
 		PostTransition:   th.PostRefreshHost(statusInfoAbortingDueClusterErrors),
 	})
 
+	// Time out while host installation
+	sm.AddTransition(stateswitch.TransitionRule{
+		TransitionType: TransitionTypeRefresh,
+		SourceStates: []stateswitch.State{
+			stateswitch.State(models.HostStatusInstallingInProgress)},
+		Condition:        stateswitch.And(th.HasInstallationTimedOut, stateswitch.Not(th.HasClusterError)),
```

Comment:
why do you need `th.HasClusterError` if cluster is already in error? 
I don't see how cluster can be in error and in progress?